test/unit_test/passes/onnx/test_perf_tuning.py

```diff
@@ -22,6 +26,36 @@ def test_ort_perf_tuning_pass(config, tmp_path):
     p.run(input_model, None, output_folder)
 
 
+@patch("olive.passes.onnx.OrtPerfTuning._run_for_config")
+@pytest.mark.parametrize(
+    "config",
+    [
+        {},
+        {"providers_list": ["CPUExecutionProvider", "CUDAExecutionProvider"], "device": "gpu"},
+    ],
+)
+def test_ort_perf_tuning_with_customized_configs(mock_run, config):
+    # setup
+    input_model = get_onnx_model()
+    p = create_pass_from_dict(OrtPerfTuning, config, disable_search=True)
+
+    # execute
+    p.run(input_model, None, None)
+
+    # assert
+    if "providers_list" not in config:
+        assert mock_run.call_args.args[2]["providers_list"] == ["CPUExecutionProvider"], (
+            "providers_list is not set correctly as ['CPUExecutionProvider'] by default when"
+            " user does not specify it"
+        )
+    if "device" not in config:
+        assert (
+            mock_run.call_args.args[2]["device"] == "cpu"
+        ), "device is not set correctly as cpu by default when user does not specify it"
```

Comment:
should it be clear?
if config:
     xxx
else:
    for k, v in config.items():

Reply:
There is the case config is not None but `providers_list` and `device` are not set.
The `providers_list` and `device` are required no matter the config is none or not.